WarOfEmpires.Domain/Players/Player.cs

```diff
@@ -36,6 +36,7 @@ public class Player : AggregateRoot {
         public virtual int BankTurns { get; protected set; } = 6;
         public virtual int Stamina { get; protected set; } = 100;
         public virtual bool HasUpkeepRunOut { get; protected set; } = false;
+        public virtual int Rank { get; protected set; } = int.MaxValue;
```

Comment:
Question: Does this mean that new players would initially be max ranked, until their rank is processed the first time?

Reply:
Bottom ranked; 1 is the max